BLMM/lib/fileio.py

```diff
@@ -517,3 +522,98 @@ def pracNumVoxelBlocks(inputs):
 
   # Return number of voxel blocks
   return(nvb)
+
+def cluster_detection(clustertype):
+        # --------------------------------------------------------------------------------
+        # Set up cluster
+        # --------------------------------------------------------------------------------
+    with open(os.path.join(os.path.expanduser("~"), ".config/dask/jobqueue.yaml"), "r") as stream:
+        data = yaml.load(stream, Loader=yaml.FullLoader)
+    if data is None:
+        raise ValueError('Please add a cluster congfiguration file in your ~/.config/dask/jobqueue.yml'
+                         ' We do not take care of dask setup so please ask your IT department for help')
+    else:
+
+        if clustertype is None:
+            if len(data["jobqueue"]) > 2:
+                raise ValueError("You have 2 cluster congfigurations in your ~/.config/dask/jobqueue.yml."
+                                 "Please specify 'clusterType' in the inputs yaml.")
+            # Check if we are using a HTCondor cluster
+            else:
+                clustertype = data["jobqueue"].keys(0)
+        if "memory" in data["jobqueue"][clustertype].keys():
+            nodemem = data["jobqueue"][clustertype]["memory"]
+        else:
+            nodemem = 0
+        nodemem = parse_bytes(nodemem)
+        if nodemem < 2 ** 10 and nodemem != 0:
```

Comment:
Since we are not imposing memory parameters I think issuing a warning is the best way to go